src/Regex.php

```diff
@@ -43,6 +43,7 @@ public static function match(string $pattern, string $subject, int $flags = 0, i
      */
     public static function matchStrictGroups(string $pattern, string $subject, int $flags = 0, int $offset = 0): MatchStrictGroupsResult
     {
+        // @phpstan-ignore composerPcre.maybeUnsafeStrictGroups
```

Comment:
I wonder why we need the ignore here. since line 47 does not contain a pattern known at static analysis time, I think it will not be analyzed by the RegexShapeMatcher